doc/users/project/mission.rst

```diff
@@ -0,0 +1,29 @@
+Matplotlib Mission Statement
+============================
+
+The Matplotlib developer community develops, maintains, and supports Matplotlib
+and its extensions to provide plotting tools for the Scientific Python
+Ecosystem.  Our success is based on how effectively we enable our users, across
```

Comment:
This reads like we are saying we are successful, rather than what I think you mean, which is that we consider it a success if we can do these things.   Maybe 

```suggestion
Ecosystem.  We judge our success based on how effectively we enable our users, across
```

Reply:
That is indeed what I meant.  This wording is better.